kumascript/macros/EmbedLiveSample.ejs

```diff
@@ -15,12 +15,19 @@
 //
 // See also : LiveSampleLink
 
-// We have deprecated the use of the $4 to $6 parameters
-// If one of this parameter is not empty, we raise a MacroDeprecationError flaw
-// When there will be 0 use left on translated-content (May 2022: 203 occurrences for $4),
+// We have deprecated the $3, $4, and $5 parameters.
+// If a non-empty value is provided for a deprecated parameter, we raise a MacroDeprecationError flaw
+// When there are 0 uses in translated-content,
 // we will be able to simplify this macro (and likely the whole of MDN).
-if ($4 !== "" || $5 !== "") {
-  mdn.deprecatedParams("The fourth and fifth parameters of 'EmbedLiveSample' are deprecated");
+// (May 2022: 203 occurrences for $4)
```

Comment:
Should we remove this line, as it's probably outdated?
```suggestion
```

Reply:
I'm fine with that, let's get rid of it